framework/src/base/Moose.C

```diff
@@ -893,6 +905,7 @@ addActionTypes(Syntax & syntax)
 "(add_postprocessor)"
 "(add_vector_postprocessor)"
 "(add_aux_kernel, add_bc, add_damper, add_dirac_kernel, add_kernel, add_dg_kernel, add_scalar_kernel, add_aux_scalar_kernel, add_indicator, add_marker)"
+"(add_control)"
```

Comment:
It needs to be last, when you call getControllableParam it accesses the objects InputParameters, since it pulls a reference it needs to be in the initialization list. So the objects being controlled must exist, thus putting it here makes sure everything exists.